setup.py

```diff
@@ -98,5 +98,8 @@ def build_extension(self, ext):
     ),
     package_data={"": ["*.yaml", "*.pf"]},
     include_package_data=True,
-    install_requires=["pyyaml"],
+    install_requires = [
+        "pyyaml",
+        "seisbench==0.4.1" if sys.version_info < (3, 9) else "seisbench>0.4.1"
+    ]
```

Comment:
Is this still needed? I thought the new setup just used the dependency from the requirement.txt. I would think the pyyaml one was a relic from previous versions.

Reply:
sure